ui/app/models/pki/issuer.js

```diff
@@ -42,6 +48,59 @@ export default class PkiIssuerModel extends PkiCertificateBaseModel {
   })
   uriSans;
 
+  @attr('string') issuerName;
+
+  @attr({
+    label: 'Leaf notAfter behavior',
+    subText:
+      'What happens when a leaf certificate is issued, but its NotAfter field (and therefore its expiry date) exceeds that of this issuer.',
+    dockLink:
+      'https://github.com/hashicorp/vault/blob/main/website/content/api-docs/secret/pki.mdx#update-issuer',
+    editType: 'yield',
+    valueOptions: ['err', 'truncate', 'permit'],
+  })
+  leafNotAfterBehavior;
+
+  @attr({
+    label: 'Usage',
+    subText: 'Allowed usages for this issuer. It can always be read',
+    editType: 'yield',
+    valueOptions: [
+      { label: 'Issuing certificates', value: 'issuing-certificates' },
+      { label: 'Signing CRLs', value: 'crl-signing' },
+      { label: 'Signing OCSPs', value: 'ocsp-signing' },
+    ],
+  })
+  usage;
+
+  @attr('string', {
+    label: 'Manual chain',
+    subText:
+      "An advanced field useful when automatic chain building isn't desired. The first element must be the present issuer's reference.",
+    editType: null,
```

Comment:
I haven't seen us use `editType: null' before - is the alternative just leaving it out of the object all together?

Reply:
Ahh good catch! This is a leftover from when I was trying to use openAPI and also override some of the attributes. I'll go ahead and remove it since it's not needed.